### DISCUSSION TITLE
How to Manually Change Filament During Print

### DISCUSSION THREAD

**Discussion by jmorganau** [bassamanator/Sovol-SV06-firmware#151] [Q&A]
Hi, bassamanator, and thanks for putting this repo together! I've been very happily running my printer per your instructions and code for over a month now. 

I don't have a filament sensor, and today I tried changing to a new spool mid-print for the first time which resulted in a large offset in the new layers printed.  I didn't see specific instructions in the README for manually changing the filament mid-print, but I thought I followed the same sequence you indicated for a filament sensor tripping.

When the spool was low, I issued an M600 macro from the dashboard. The print head parked itself at the front of the bed, and I used the dashboard to retract the filament and then removed it. I then inserted the end of the new spool, extruded 50mm, and hit resume.

Did I do anything wrong in that process, or do you see any other reason the new layers would be offset from the rest of the print? I do not think I could have moved any of the axes during the filament change.

**Comment by bassamanator** [comment]
There should be no offset at all. The process you used is fine, though not how I normally do things, but again, it's fine.

Check the how to section in the everything sv06 repo for a video on how to do a colour change. Also read the faq section.
